en/guide/modules.md

```diff
@@ -151,12 +130,12 @@ module.exports = {
     '@nuxtjs/axios'
   ],
 
-  // axios module is aware of this by using this.options.axios
+  // axios module is aware of this by using `this.options.axios`
```

Comment:
Tick for code, even in comment (because of Markdown is a language for plain text)